src/common/server.ts

```diff
@@ -175,7 +175,7 @@ async function createNativeServer(
 
   logger.info(`Found Ruff ${versionToString(ruffVersion)} at ${ruffBinaryPath}`);
 
-  if (!ruffBinaryPath.endsWith("red_knot")) {
+  if (!ruffBinaryPath.endsWith("red_knot") && !ruffBinaryPath.endsWith("red_knot.exe")) {
```

Comment:
I suggest extracting this into a variable: `let is_red_knot = ....` and then use that variable in the checks instead.